linetools/guis/xspecgui.py

```diff
@@ -75,6 +75,18 @@ def __init__(self, ispec, parent=None, zsys=None, norm=None, exten=None,
                                                 zsys=zsys, norm=norm, exten=exten,
                                                 screen_scale=self.scale,
                                                  rsp_kwargs=rsp_kwargs, **kwargs)
+        # Reset redshift from spec
+        if zsys is None:
+            if hasattr(self.spec_widg.spec, 'z'):
+                self.pltline_widg.setz(str(self.spec_widg.spec.z[self.spec_widg.select]))
+        # Auto set line list if spec has proper object type
+        if hasattr(self.spec_widg.spec, 'stypes'):
+            if self.spec_widg.spec.stypes[self.spec_widg.select].lower() == 'galaxy':
+                self.pltline_widg.llist = ltgu.set_llist('Galaxy',in_dict=self.pltline_widg.llist)
+                self.pltline_widg.llist['Plot'] = True
+                idx = self.pltline_widg.lists.index(self.pltline_widg.llist['List'])
+                self.pltline_widg.llist_widget.setCurrentRow(idx)
+        #
```

Comment:
Loading up the GUI automatically selects the correct linelist for 'Galaxy', but the lines do not immediately show.  Instead, I have a click on a different linelist in this widget then then back to Galaxy.  At that point, I see the galaxy lines plotted.  This is quite possibly a Qt 'idiosyncracy' and perhaps works properly on other platforms.  Are the lines marked when you open up the GUI on your machine?

Reply:
nope, it is an idiosyncracy..